packages/astro/astro-jsx.d.ts

```diff
@@ -9,7 +9,7 @@
  * Adapted from React’s TypeScript definition from DefinitelyTyped.
  * @see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts
  */
-declare namespace astroHTML.JSX {
+namespace astroHTML.JSX {
```

Comment:
Is the `declare` removal caused by prettier?

Reply:
Yes. The files changed are caused by running prettier locally (which is good, it seems we caught a bug?)